javascript/presigned-url-example.ts

```diff
@@ -0,0 +1,140 @@
+import {MomentoSigner, CacheOperation} from '@gomomento/sdk';
+import fetch, {Response} from 'node-fetch';
+
+const signingKey = process.env.MOMENTO_SIGNING_KEY;
+if (!signingKey) {
+  throw new Error('Missing required environment variable MOMENTO_SIGNING_KEY');
+}
+const signingEndpoint = process.env.MOMENTO_ENDPOINT || '';
+if (signingEndpoint === '') {
+  throw new Error('Missing required environment variable MOMENTO_ENDPOINT');
+}
+
+let cacheName: string;
+if (process.env.CACHE_NAME) {
+  cacheName = process.env.CACHE_NAME;
+} else {
+  cacheName = 'default-cache';
+  console.log(
+    `Environment variable CACHE_NAME not set. Using cache name '${cacheName}'.`
+  );
+}
+const cacheKey = 'MyCacheKey';
+const cacheValue = 'MyCacheValue';
+const expiryEpochSeconds = Math.floor(Date.now() / 1000) + 10 * 60; // 10 minutes from now
+const objectTtlSeconds = 180;
+
+const main = async () => {
```

Comment:
style nits.

1. why declare this as a `const` rather than just use the normal `function` keyword?
2. it seems weird to me to have this `main` function that presumably encapsulates all of the logic for the main entrypoint to the script, but then have so many lines of code defining global variables and such outside of the main function.  Could we just move those inside `main`?

Reply:
I copied this from https://github.com/momentohq/client-sdk-examples/blob/54435cefb86cf04b46e944a5e2fe32a74d1770ba/javascript/index.ts#L60-L66 so honestly I don't have a good answer for this. But yeah, I can just move this into `main`.